### ISSUE TITLE
Tera Logo

### ISSUE THREAD

**Issue by frustak** [Keats/tera#599]
I think Tera needs some kind of logo, so it would be able to have a file icon in editors like VSCode (with extensions probably), it makes it much easier to distinguish files.

**Comment by uncenter** [comment]
Definitely! Would be nice to have a logo to go off of for file icon packs :)

I'm not sure where exactly the name for Tera came from, but it looks like the "tera-" prefix (like in terabyte) comes from the Greek word "Τέρας" ("teras") meaning monster? Could be something to start with for a mascot/logo... 🤔 

![Screenshot 2024-05-12 at 22 00 50 (Arc)](https://github.com/Keats/tera/assets/47499684/2323941a-693d-4bb2-a0d5-a86ea18f551d)

**Comment by Keats** [comment]
> I'm not sure where exactly the name for Tera came from,

It's kind of a synonym of Jinja. Jinja is a shintoist building and Tera a buddhist building. I created Tera when I was living in Japan and there was some お寺 (otera) signs close to my flat.

**Comment by uncenter** [comment]
What do those kind of buildings look like? I guess that would make the most sense right, like the outline of the building structure with a single solid color would be simple and look nice :)

**Comment by uncenter** [comment]
Reading through https://en.wikipedia.org/wiki/Buddhist_temples_in_Japan, there are some interesting takeaways. The Jinja2 logo is of a [torii](https://en.wikipedia.org/wiki/Torii), the gate at the entrance to a Shinto shrine; maybe even based on https://commons.wikimedia.org/wiki/File:Itsukushima_Gate.jpg, the first image shown on the Wikipedia page for torii:

![Jinja2 logo](https://github.com/user-attachments/assets/dcd3f925-73a5-415b-8d74-ebf32d18ac85)
![1024px-Itsukushima_Gate](https://github.com/user-attachments/assets/0f25210f-429e-4145-9574-a35994847e82)

Tera, being Buddhist as opposed to Shintoist, has a [rōmon](https://en.wikipedia.org/wiki/R%C5%8Dmon) instead.

![image](https://github.com/user-attachments/assets/a0e05330-c9be-4c5d-8d25-e4e129b8f29e)

I think these look different enough actually, with the large roof of the rōmon being quite distinguishing.

**Comment by Keats** [comment]
Closing as part of the Tera v2 release. If this is still relevant in v2, please reopen — see the 
  migration guide: https://github.com/Keats/tera/blob/master/MIGRATION.md
